calour/experiment.py

```diff
@@ -341,6 +377,42 @@ def reorder(self, new_order, axis=0, inplace=False):
                 exp.feature_metadata = exp.feature_metadata.iloc[new_order, :]
         return exp
 
+    def chain(self, steps=[], inplace=False, **kwargs):
```

Comment:
what is chain() used for?
didn't see any reference in the pr
do you have a use for it?